<launch>
    <node pkg="object_pose" type="pose_estimation" name="pose_estimation">
        <param name="/location_meshes"
value="/home/fabian/Documents/work/transforms/src/ObjectPose/data/meshes.txt"/>
        <param name="/segmented" value="/segmented"/>
        <param name="/estimated_poses" value="/geometric"/>
</node>
</launch>
